style.css

```diff
@@ -132,51 +381,17 @@ header {
   }
 
   .pro-fa {
-    display: none;
+    display: none !important;
```

Comment:
Don't use ```!important``` in the css files

Reply:
Wow! Noted sir, we will try and Implement all the bugs you spotted to make the required changes.👍

Reply:
write me on slack if you have any question